documentation/source/user_section/installation.rst

```diff
@@ -96,8 +96,120 @@ In the context of SCT, it can be used:
+   # Launch interactive mode (command-line inside container)
+   docker run -it ubuntu
+   # Now, inside Docker container, install dependencies
+   apt-get update
+   yes | apt install git curl bzip2 libglib2.0-0 gcc
+   # Note for above: libglib2.0-0 is required by PyQt
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/ubuntu:ubuntu16.04
+
+CentOS7-based installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+   # Start from the Terminal
+   docker pull centos:centos7
+   # Launch interactive mode (command-line inside container)
+   docker run -it centos:centos7
+   # Now, inside Docker container, install dependencies
+   yum install -y which gcc git curl
+   # Install SCT
+   git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+   cd sct
+   yes | ./install_sct
+   export PATH="/sct/bin:${PATH}"
+   # Test SCT
+   sct_testing
+   # save the state of the container. Open a new Terminal and run:
+   docker ps -a  # list all containers
+   docker commit <CONTAINER_ID> <YOUR_NAME>/centos:centos7
+
+
+Using GUI Scripts (Optional)
+============================
+
+In order to run scripts with GUI you need to allow X11 redirection.
+First, save your Docker image:
+
+1. Open another Terminal
+2. List current docker images
+
+.. code:: bash
+
+   docker ps -a
+
+3. Save container as new image
+
+.. code:: bash
+
+   docker commit <CONTAINER_ID> <YOUR_NAME>/<DISTROS>:<VERSION>
+
+For macOS and Linux users
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create an X11 server for handling display:
+
+1. Install XQuartz X11 server.
+2. Check ‘Allow connections from network clientsoption inXQuartz\`
+   settings.
+3. Quit and restart XQuartz.
+4. In XQuartz window xhost + 127.0.0.1
+5. In your other Terminal window, run:
+
+   -  On macOS:
+      ``docker run -e DISPLAY=host.docker.internal:0 -it <CONTAINER_ID>``
+   -  On Linux:
+      ``docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix <CONTAINER_ID>``
+
+For Windows users
+~~~~~~~~~~~~~~~~~
+
+1. Install Xming
+2. Connect to it using Xming/SSH:
+
+  - If you are using Docker Desktop, please download and run (double click) the following script: :download:`sct-win.xlaunch<../../../contrib/docker/sct-win.xlaunch>`.
+  - If you are using Docker Toolbox, please download and run the following script instead: :download:`sct-win_docker_toolbox.xlaunch<../../../contrib/docker/sct-win_docker_toolbox.xlaunch>`
+  - If this is the first time you have done this procedure, the system will ask you if you want to add the remote PC (the docker container) as trust pc, type yes. Then type the password to enter the docker container (by default sct).
+
+**Troubleshooting:**
+
+| If there are no new open windows:
```

Comment:
any reason for the "|"?  The list does not render correctly:
![image](https://user-images.githubusercontent.com/2482071/97377142-ac02f200-1895-11eb-80fd-b84ffa98b3f4.png)